src/main/markdown/doc/latest/RefJreEmulation.md

```diff
@@ -102,7 +108,7 @@ Note that in some cases, only a subset of methods is supported for a given type.
   <dd>Byte(byte), Byte(String), compare(byte, byte), decode(String), hashCode(byte), parseByte(String), parseByte(String, int), toString(byte), valueOf(byte), valueOf(String), valueOf(String, int), byteValue(), compareTo(Byte), doubleValue(), equals(Object), floatValue(), hashCode(), intValue(), longValue(), shortValue(), toString()</dd>
 
   <dt><a href="https://docs.oracle.com/javase/8/docs/api/java/lang/CharSequence.html">CharSequence</a></dt>
-  <dd>charAt(int), length(), subSequence(int, int), toString(), chars(), $isInstance(HasCharSequenceTypeMarker)</dd>
```

Comment:
Oh, looks like we had that issue already with those internal methods!

(since 2.8.0, present in the last two commits that **I** made :facepalm:)

Reply:
Yep, but since you can't access them as they aren't part of the JRE, it does seem reasonable to remove them (and possibly update tooling to look for these j2cl-specific methods and avoid them to begin with.